tests/snuba/search/test_backend.py

```diff
@@ -922,7 +922,7 @@ def __eq__(self, other):
         assert query_mock.call_args == mock.call(
             orderby=["-priority", "issue"],
             aggregations=[
-                ["toUInt64(plus(multiply(log(times_seen), 600), last_seen))", "", "priority"],
+                ["toUInt64(plus(multiply(log(times_seen), 600), `last_seen`))", "", "priority"],
```

Comment:
Why are there backticks in here?

Reply:
It came in from the groups dataset aggregations, which define this using `events.last_seen`, and without the ticks gives an error.

Now that they are more decoupled though, I can remove the ticks from this aggregation.